firebase-installations/src/main/java/com/google/firebase/installations/local/IidStore.java

```diff
@@ -38,17 +40,23 @@
   private static final String IID_SHARED_PREFS_NAME = "com.google.android.gms.appid";
   private static final String STORE_KEY_PUB = "|S||P|";
   private static final String STORE_KEY_ID = "|S|id";
+  private static final String STORE_KEY_TOKEN = "|T|";
+  private static final String DEFAULT_SCOPE = "|*";
+  private static final String JSON_TOKEN_KEY = "token";
```

Comment:
I have a feeling that you want to create some constant "TOKEN_STORAGE_DELIMITER" or something and add that String between all these constant values.
Maybe also add a constant "TOKEN_STORAGE_KEY_FORMAT", something like this:
"%s|%s|%s" or whatever the right format is and then later fill the placeholders with the actual values.